data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,58 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"Alright, I'm interested. Where is this 'soiree' taking place?"`
+				`	"I'm not a fan of plays, sorry."`
+					decline
+			`	You follow her around the central portion of the city, to a building that from the outside looks like some form of auditorium, where some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, young lady," one of them says to your guide, as she shows him a touchpad displaying two tickets. He verifies them, and allows you two to enter. The inside reminds you of a human opera house, with a large stage in front of the dozens of rows of seats, and six levels of balconies on both sides. The Arach leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arachi musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage, to broadcast the event for those who didn't have the chance to attend it live.`
+			`	As the rows and balconies fill up with more guests, you notice a few Kimek and Saryds amidst the masses of Arachi, most of them accompanying an Arach patron, but a few seemingly being patrons themselves, coming in on their own. A great many of them seem to take notice of you as you look around, but your patron studiously ignores the attention. Once all seats are filled, the doors are closed, and an Arach comes to the stage. "Welcome you, esteemed guests, to this celebration of saga and song, our theater does," he says. "Make you wait any longer, I shall not, so for our first play, a timeless classic dear to all, I present: Moonstruck."`
+			`	He quickly leaves the stage as the curtains open to reveal a rocky wall, with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen, and a large sack hanging from his back. He picks rocks from the corners of the wall, throwing most to the ground, and placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side, where the young miner is shown outside the mine, under the night sky, smelting the ore he collected. As he looks to the top of the stage, his back facing the audience, the accordions grow quieter, and a slow harp starts to play, as a full moon comes into view, at the top of the background sky.`
```

Comment:
```suggestion
			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. He picks rocks from the corners of the wall, throwing most to the ground and placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side, where the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage, his back facing the audience, the accordions grow quieter, and a slow harp starts to play as a full moon comes into view at the top of the background sky.`
```
Just some commas and a couple small things: avoiding using `hanging` twice in the same sentence and taking `outside the mine` out (sort of redundant since `under the night sky` would imply we're outside the mine now anyways).